decidim-assemblies/lib/decidim/assemblies/participatory_space.rb

```diff
@@ -8,6 +8,8 @@
     Decidim::Assemblies::OrganizationAssemblies.new(organization).query
   end
 
+  participatory_space.permissions_class_name = "Decidim::Assemblies::Permissions"
```

Comment:
Should we default to a pass-through permissions class?

Reply:
Something that simply returns `true`, you mean?

Reply:
Yeah!

Reply:
Done! It's called `Decidim::DefaultPermissions`